mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -1227,6 +1302,18 @@ func expandInstanceLifecyclePolicy(configured []interface{}) *compute.InstanceGr
 	return instanceLifecyclePolicy
 }
 
+<% unless version == "ga" -%>
+func expandStandbyPolicy(d *schema.ResourceData) *compute.InstanceGroupManagerStandbyPolicy {
+	standbyPolicy := &compute.InstanceGroupManagerStandbyPolicy{}
+	for _, sp := range d.Get("standby_policy").([]any) {
+		spData := sp.(map[string]any)
+		standbyPolicy.InitialDelaySec = int64(spData["initial_delay_sec"].(int))
```

Comment:
I think we must ForceSendFields in case we're changing this value to 0.

Reply:
Feel free to include this in a test to check.

Reply:
Check please.

Reply:
looks great, thanks!